services/review_manager.js

```diff
@@ -17,3 +17,27 @@ exports.post_review_process = async(req,resp,input_response)=>{
   resp.send(input_response)
 }
 
+exports.pre_process_check_eligibility = async(req,resp)=>{
+  return {"coach_id":req.query.coach_id,"user_id":req.user.user_id}
```

Comment:
if coach_id is not present in coach throw 400 Error code with message as "coach id not present"